.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -119,18 +119,22 @@ jobs:
           mkdir Kokkos
           cp -rf /root/Kokkos_install/${{ matrix.exec_model }}/* Kokkos/
 
-      - name: Install dependencies (CentOS)
-        if: ${{ (matrix.container_img == 'quay.io/pypa/manylinux2014_x86_64') }}
+      - name: Install dependencies (AlmaLinux)
+        if: ${{ (matrix.container_img == 'quay.io/pypa/manylinux_2_28_x86_64') }}
         run: |
-          # Reduce wait time for repos not responding
-          cat /etc/yum.conf | sed "s/\[main\]/\[main\]\ntimeout=5/g" > /etc/yum.conf
-          yum update -y && yum install -y docker
+          yum update -y
+          yum install yum-utils -y
+          yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+          yum install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+          systemctl enable --now docker
 
       - name: Install cibuildwheel
         run: python3.9 -m pip install cibuildwheel~=2.16.0
 
       - name: Build wheels
         env:
+          # CIBW_CONTAINER_ENGINE: "docker"
```

Comment:
We could probably install `podman` onto the outer Ubuntu image here, and use it as a backend instead of docker. This may circumvent the systemd error we see on some of the CI runners when spinning up the builder

Reply:
I'm exploring this approach.